prow/github/types.go

```diff
@@ -575,10 +575,12 @@ type Content struct {
 
 // Team is a github organizational team
 type Team struct {
-	ID          int    `json:"id,omitempty"`
-	Name        string `json:"name,omitempty"`
-	Description string `json:"description,omitempty"`
-	Privacy     string `json:"privacy,omitempty"`
+	ID           int    `json:"id,omitempty"`
+	Name         string `json:"name,omitempty"`
+	Description  string `json:"description,omitempty"`
+	Privacy      string `json:"privacy,omitempty"`
+	Parent       *Team  `json:"parent,omitempty"`
```

Comment:
Maybe note with a comment that this is only relevant for responses

Reply:
Done.